script/playbackstrategy/msestrategy.js

```diff
@@ -68,10 +71,24 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
 
       function onBuffering () {
         isEnded = false;
-        publishMediaState(MediaState.WAITING);
+        if (!isSeeking || !publishedSeekEvent) {
```

Comment:
Assuming you will clean up this debugging before merging? If so, should this be a draft PR?

Reply:
Yes, It's only added for testing purposes. 

Though we can't change a PR to Draft PR.